app/learning.js

```diff
@@ -7,51 +7,86 @@
  * Main script for initiating the learning app.
  */
 
-define(['jquery', 'bootstrap', 'app/config', 'app/database', 'app/messages', 'app/question', 'app/timer','app/database'], function ($, bootstrap, config, db, messages, question, timer,db) {
-  timer.startTimer(".timer", config.constant("TIME_LIMIT"));
+/*jshint esversion: 6 */
+define(['jquery', 'app/lang', 'app/string', 'bootstrap', 'app/config', 'app/database', 'app/messages', 'app/question', 'app/timer', 'app/ready', 'app/user', 'app/time'], function ($, lang, string, bootstrap, config, db, messages, questions, timer, ready, user, time) {
+
+  const THOUSAND = 1000;
```

Comment:
Nope, nope, nope

Take a look at this [post](http://thedailywtf.com/articles/Avoiding-Magic-Constants)